node/src/command.rs

```diff
@@ -349,10 +349,15 @@ impl SubstrateCli for Cli {
 
     fn native_runtime_version(chain_spec: &Box<dyn sc_service::ChainSpec>) -> &'static RuntimeVersion {
         match chain_spec.runtime_name().as_str() {
+            #[cfg(feature = "phala-native")]
```

Comment:
Better to keep the default runtime compiled (phala, khala)?

Reply:
All runtimes are pluggable (controlled by feature gate like this),
default runtimes are enabled in `Cargo.toml`, not here